cscs-checks/apps/vasp/vasp_check.py

```diff
@@ -7,130 +7,124 @@
 import reframe.utility.sanity as sn
 
 
+@rfm.simple_test
 class VASPCheck(rfm.RunOnlyRegressionTest):
-    scale = parameter(['small', 'large'])
-    variant = parameter(['maint', 'prod'])
+    descr = f'VASP check '
     modules = ['VASP']
     executable = 'vasp_std'
-    keep_files = ['OUTCAR']
     extra_resources = {
         'switches': {
             'num_switches': 1
         }
     }
+    keep_files = ['OUTCAR']
     strict_check = False
+    use_multithreading = False
+    tags = {'maintenance', 'production'}
     maintainers = ['LM']
-    tags = {'scs'}
-
-    @run_after('init')
-    def setup_by_system(self):
-        if self.current_system.name in ['eiger', 'pilatus']:
-            self.valid_prog_environs = ['cpeIntel']
-        else:
-            self.valid_prog_environs = ['builtin']
 
-        self.tags |= {
-            'maintenance' if self.variant == 'maint' else 'production'
+    num_nodes = parameter([6, 16], loggable=True)
+    references = {
+        6: {
+            'sm_60': {
+                'dom:gpu': {'time': (56.0, None, 0.10, 's')},
+                'daint:gpu': {'time': (65.0, None, 0.15, 's')},
+            },
+            'broadwell': {
+                'dom:mc': {'time': (58.0, None, 0.10, 's')},
+                'daint:mc': {'time': (65.0, None, 0.15, 's')},
+            },
+            'zen2': {
+                'eiger:mc': {'time': (100.0, None, 0.10, 's')},
+                'pilatus:mc': {'time': (100.0, None, 0.10, 's')},
+            },
+        },
+        16: {
+            'sm_60': {
+                'daint:gpu': {'time': (55.0, None, 0.15, 's')},
+            },
+            'broadwell': {
+                'daint:mc': {'time': (55.0, None, 0.15, 's')},
+            },
+            'zen2': {
+                'eiger:mc': {'time': (100.0, None, 0.10, 's')},
+                'pilatus:mc': {'time': (100.0, None, 0.10, 's')}
+            }
         }
-
-    @sanity_function
-    def assert_reference(self):
-        force = sn.extractsingle(r'1 F=\s+(?P<result>\S+)',
-                                 self.stdout, 'result', float)
-        return sn.assert_reference(force, -.85026214E+03, -1e-5, 1e-5)
+    }
 
     @performance_function('s')
     def time(self):
```

Comment:
I would probably call that `elapsed_time`.

Reply:
I have modified the performance function name accordingly.